### PR TITLE
Ask contributors to use `tox` for final notebook execution.

### LINKED ISSUE #1239: More explicitly direct people to run notebooks with `tox`
The majority of our authors use VSCode, which will by default generate PNGs rather than the superior SVGs. We should tell them how to set up VSCode to use SVGs instead.

This is probably a global config you set for Jupyter?

This change would help avoid the need for https://github.com/Qiskit/documentation/pull/1226.

### PR DESCRIPTION
**Pull request by frankharkins**
Closes #1239

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,9 @@ This staging environment can be useful to see how the docs are rendering before
 
 ## Execute notebooks
 
+Before submitting a new notebook or code changes to a notebook, you must run the notebook using `tox -- --write <path-to-notebook>` and commit the results.
+This means we can be sure all notebooks work and that users will see the same results when they run using the environment we recommend.
```
Comment (Eric-Arellano) [inline README.md:107]:
```suggestion
Before submitting a new notebook or code changes to a notebook, you must run the notebook using `tox -- --write <path-to-notebook>` and commit the results. (If the notebook submits jobs, also use the argument `--submit-jobs`). This means we can be sure all notebooks work and that users will see the same results when they run using the environment we recommend.
```

**Review by Eric-Arellano — COMMENTED** [review]
Hm I guess we already say some of this text in the final paragraph of this section. Maybe remove some of that text? I think your change makes sense to explain the motivation for testing notebooks right away, and only then explain how to do it.

File: README.md [hunk 2]
```diff
@@ -130,20 +137,20 @@ You also need to install a few system dependencies: TeX, Poppler, and graphviz.
 > [!NOTE]
 > If your notebook submits hardware jobs to Qiskit Runtime, you must add it to
 > [`scripts/nb-tester/notebooks.toml`](scripts/nb-tester/notebooks.toml). If it
-> can be run with simulators, i.e., the circuit is not too large, add it to `notebooks_that_submit_jobs`.
-> Otherwise, add it to `notebooks_no_mock`.
+> can be run with simulators, i.e., the circuit is not too large, add it to
+> `notebooks_that_submit_jobs`. Otherwise, add it to `notebooks_no_mock`.
```
[the remark this answers is not in the document]
Comment (frankharkins) [reply README.md]: Oops thanks, should be fixed now

**Review by Eric-Arellano — APPROVED** [review]
Looks great!

**Pull request merged**